.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please provide the following information and someone from @imgix/imgix-sdk-team will respond to your issue shortly.
+
+**Before you submit:**
+
+- [ ] Please search through the [existing issues]([existing issues](https://github.com/imgix/imgix-php/issues?utf8=%E2%9C%93&q=is%3Aissue)) to see if your question has already been discussed.
```

Comment:
Is this a typo?

```
[existing issues]([existing issues](https://github.com/imgix/imgix-php/issues?utf8=%E2%9C%93&q=is%3Aissue))
```

Reply:
Yeah good catch, also the file was supposed to be `question.md` but still has the generic template outline. I'll fix both.